src/main/scala/com/mozilla/telemetry/pings/Ping.scala

```diff
@@ -392,6 +392,7 @@ object SendsToAmplitude {
   def apply(message: Message): SendsToAmplitude = {
     message.fieldsAsMap.get("docType") match {
       case Some("focus-event") => FocusEventPing(message)
+      case Some("rocket-event") => RocketEventPing(message)
```

Comment:
This won't work, unfortunately. The messages being sent from Rocket do have a docType of `focus-event`. If you match on both the `docType` and the `appName`, that could do the trick.